tap_github/repository_streams.py

```diff
@@ -2625,3 +2625,131 @@ class TagsStream(GitHubRestStream):
         th.Property("tarball_url", th.StringType),
         th.Property("node_id", th.StringType),
     ).to_dict()
+
+
+class DeploymentsStream(GitHubRestStream):
+    """A stream dedicated to fetching deployments in a repository."""
+
+    name = "deployments"
+    path = "/repos/{org}/{repo}/deployments"
+    primary_keys: ClassVar[list[str]] = ["repo", "org", "node_id"]
```

Comment:
Shouldn't `node_id` be sufficient to globally and uniquely identify a deployment?

```suggestion
    primary_keys: ClassVar[list[str]] = ["node_id"]
```

https://docs.github.com/en/graphql/guides/using-global-node-ids

Reply:
yep updated both suggestion